### DISCUSSION TITLE
very poor hardware detection Elementary 8

### DISCUSSION THREAD

**Discussion by Fatjoeny** [elementary/triage#645] [Q&A] [answered]
When I installed the new Elementary OS 8 on my notebook, it did NOT recognize the different USB stick WLAN adapters!!!!! Because the internal WLAN adapter is broken, I have to connect WLAN USB stick adapters to several notebooks, but unfortunately this operating system does not recognize them at all!!!!!.
It always says in the WLAN symbol at the bottom right when you click on it (YOUR HARDWARE IS PHYSICALLY SET TO OFF WITH A HARDWARE SLIDING CONTROL), which is not true at all, because 99 percent of WLAN USB sticks do not have a physical sliding control to turn them off!!!!!.
I sincerely ask that you manage to recognize a lot of WLAN USB stick adapters during installation without you having to do anything yourself, THANK YOU :-)

Best Regards

**Comment by teamcons** [comment, marked as answer]
Hello

Does Ubuntu usually recognizes them ?
elementary OS pretty much just piggybacks on Ubuntu kernel and stack, former which is pretty much patched Linux kernel.
Thats the one responsible for hardware detection afaik. Any fix for Ubuntu is a fix for elementary OS

Please do not abuse exclamation points and all caps.

**Reply by davidmhewitt** [reply]
Closing this discussion as OP has stated this hardware doesn't work on Ubuntu either and we use the same drivers.

**Comment by Fatjoeny** [comment]
No, Ubuntu is too stupid. It doesn't know the whole hardware USB stick WLAN
adapter either. It would be nice if they could somehow make it recognize
the only Linux that I have tested. Of the 50 Linux systems tested, only
anti x is the very best that recognizes everything.

Stella and Charlie ***@***.***> schrieb am Fr., 6. Dez. 2024,
12:09:

**Comment by teamcons** [comment]
Im not sure why antix recognized anything, but if Ubuntu cant, then any distribution built on Ubuntu likely cant either - elementary, Mint, etc.

this has to do with the kernel shipped. Elementary doesnt do anything to it, so hardware issues have to do with upstream, not elementary.

So theres nothing that can really be done on elementary's side.

**Comment by Fatjoeny** [comment]
Wie gesagt, ist Antix die einzigste Linux System die jede Hardware
erkennt!!!!!, egal ob WLAN USB Stick Adapter oder jegliche Art von
Drucker!!!!!. Das ist das aller beste Linux System auf der Welt!!!!!.

Stella and Charlie ***@***.***> schrieb am Fr., 6. Dez. 2024,
15:44:

**Comment by Fatjoeny** [comment]
Was soll der scheiß?????.
Das Problem wurde noch lange nicht gelöst!!!!!, und sie schließen es als
gelöst ab. Da sieht mann was für komische Menschen Ihr seit!!!!!, kein
bißchen anstrengen um den Hardware Erkennungsfehler zu beheben BRAVO WEITER
SO UNSERIÖS SEIN!!!!!!!!!

David Hewitt ***@***.***> schrieb am So., 8. Dez. 2024, 13:36:

**Reply by teamcons** [reply]
1/ Please write in English.

2/ You had an answer: This is a problem with upstream. Nothing elementary OS can do here.
You also found an alternative solution you were happy with.

3/ The people answering you in Discussions are volunteers.

4/ Please reread community guidelines. This reaction is unacceptable.
